targets/TARGET_NORDIC/TARGET_NRF5/TARGET_MCU_NRF52840/PeripheralNames.h

```diff
@@ -45,8 +45,6 @@
 extern "C" {
 #endif
 
-#define STDIO_UART_TX     TX_PIN_NUMBER
```

Comment:
this is similar to what I observed https://github.com/ARMmbed/mbed-os/pull/5618 , I can close mine if this gets approved.